security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -41,6 +41,15 @@ const (
 	// The Istio secret annotation type
 	IstioSecretType = "istio.io/key-and-cert"
 
+	// NamespaceManagedLabel and NamespaceOverrideLabel contribute to determining whether or not
```

Comment:
Please also point out that `NamespaceManagedLabel` is a string (with namespace as the value) and `NamespaceOverrideLabel` is a boolean.

Reply:
fixed